### PR TITLE
Feat/make tools as packages

### PR DESCRIPTION
**Pull request by 0xArdi**
This PR adds support for having tools as custom packages.

### CODE REVIEW

**Review by angrybayblade — APPROVED** [review]

File: packages/valory/skills/task_execution/utils/task.py [hunk 1]
```diff
@@ -28,8 +28,9 @@ class AnyToolAsTask:
     def execute(self, *args: Any, **kwargs: Any) -> Any:
         """Execute the task."""
         tool_py = kwargs.pop("tool_py")
-        if "run" in globals():
-            del globals()["run"]
+        callable_method = kwargs.pop("callable_method")
```
Comment (DavidMinarsch) [inline packages/valory/skills/task_execution/utils/task.py:31]: Is this going to work with multiple packages having the same callable name?

**Review by DavidMinarsch — COMMENTED** [review]
@0xArdi one comment - ow looks good 

please check for each package who the author was. In some cases it’s not Valory

File: packages/valory/skills/task_execution/utils/task.py (hunk 1 above)
Comment (0xArdi) [reply packages/valory/skills/task_execution/utils/task.py:31]: yes

Comment (0xArdi) [reply packages/valory/skills/task_execution/utils/task.py:31]: all tools had `run` as their callable and it worked

**Review by jmoreira-valory — APPROVED** [review]

**Pull request merged**